src/Client.ts

```diff
@@ -87,6 +87,21 @@ export type NetworkOptions = {
    * SDK updates, including deprecations and required upgrades.
    */
   appVersion?: string
+  /**
+   * Skip publishing the user's contact bundle as part of Client startup.
+   *
+   * This flag should be used with caution, as we rely on contact publishing to
+   * let other users know your public key and periodically run migrations on
+   * this data with new SDK versions.
+   *
+   * Your application should have this flag set to `false` at least _some_ of the
+   * time.
+   *
+   * The most common use-case for setting this to `true` is cases where the Client
+   * instance is very short-lived. For example, spinning up a Client to decrypt
+   * a push notification.
+   */
+  skipContactPublishing: boolean
```

Comment:
Given the potential side-effects of this flag (if you always keep it on you will be unreachable on the network) I am inclined to keep the flag narrowly scoped. Something like `headlessMode` feels like it might be more confusing, and lead to someone with a Node.js app using it without understanding the consequences.

Reply:
I'm a fan of naming the flag as obviously as possible i.e. name = exactly what the flag does. I think if we start to get into a world of too many config flags, we can start consolidating into "modes".